Cython/Compiler/ModuleNode.py

```diff
@@ -2862,9 +2862,12 @@ def generate_import_star(self, env, code):
                 code.putln('else if (__Pyx_StrEq(name, "%s")) {' % name)
                 if entry.type.is_pyobject:
                     if entry.type.is_extension_type or entry.type.is_builtin_type:
+                        type_test, type_test_utility = entry.type.type_test_code(
+                            code.funcstate.scope, "o")
```

Comment:
Isn't this just `env`? (Plus, maybe, going up the scope hierarchy until we hit a function or module scope, I guess.)